### PR TITLE
V1

### PR DESCRIPTION
**Pull request by cornelsu**

### CODE REVIEW

**Comment by cornelsu** [comment]
ok

File: 03-Azure/01-02 Data/01-MicroHack-SAP-Cashflow-Prediction/SAP-Data-MicroHack.pptx
Comment (cthoenes) [inline 03-Azure/01-02 Data/01-MicroHack-SAP-Cashflow-Prediction/SAP-Data-MicroHack.pptx:1]: Seems like this change is not intended? Can you please remove changes to this file from the pr?

File: 03-Azure/01-03-Infrastructure/05_Azure_VMware_Solution/Lab/info/AVS Labs Credentials - Example.xlsx
Comment (cthoenes) [inline 03-Azure/01-03-Infrastructure/05_Azure_VMware_Solution/Lab/info/AVS Labs Credentials - Example.xlsx:1]: Seems like this change is not intended? Can you please remove changes to this file from the pr?

**Review by cthoenes — CHANGES_REQUESTED** [review]
Initial request to remove changes to unrelate files.

**Pull request closed without merging**